libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserTests.java

```diff
@@ -174,4 +196,43 @@ public void testMultipleConstructorsAnnotated() throws IOException {
             )
         );
     }
+
+    public void testMultipleMethodsAnnotated() throws IOException {
```

Comment:
Can we also have a error test for a non-static method having the annotation?

Reply:
Yes! Good call.